### PR TITLE
renaming demo to fabric-website-resources

### PR DESCRIPTION
**Pull request by kenotron**
#### Pull request checklist

- [ ] Addresses an existing issue: Fixes #0000
- [ ] Include a change request file using `$ npm run change`

#### Description of changes

renaming demo to fabric-website-resources

#### Focus areas to test

(optional)

### CODE REVIEW

**Comment by cliffkoh** [comment]
Can you try doing a full production build + try publish dryrun? (Use `rush publish`).

**Comment by cliffkoh** [comment]
From a clean state.. so `git clean -fdx` first :) It's hard to tell from a CR perspective otherwise whether you have taken care of all bases.

File: rush.json [hunk 1]
```diff
@@ -72,7 +72,7 @@
       "shouldPublish": true
     },
     {
-      "packageName": "@uifabric/demo",
+      "packageName": "@uifabric/fabric-website-resources",
       "projectFolder": "apps/demo",
```
Comment (cliffkoh) [inline rush.json]: I think the folder name should mirror the package name to avoid confusion down the road..

**Comment by kenotron** [comment]
I've tested prod build and dryrun publish.

File: rush.json (hunk 1 above)
Comment (kenotron) [reply rush.json]: I'll rename folders in another PR - I wanted to minimize the risk with this PR to unblock

**Pull request merged**